prime-router/src/testIntegration/kotlin/datatests/mappinginventory/catchall/aui/AUIExtensionTests.kt

```diff
@@ -0,0 +1,17 @@
+package gov.cdc.prime.router.datatests.mappinginventory.aui
+
+import gov.cdc.prime.router.datatests.mappinginventory.verifyHL7ToFHIRToHL7Mapping
+import org.junit.jupiter.api.Test
+
+class AUIExtensionTests {
+    @Test
+    fun `test AUI mapped to AUIExtension`() {
+        assert(
+            verifyHL7ToFHIRToHL7Mapping(
+                "catchall/aui/AUI-to-Extension",
+                // todo update the schema to OML_O21-base.yml when OML updates are merged to main
```

Comment:
Include ticket number

Reply:
Added in a link to https://github.com/CDCgov/prime-reportstream/issues/15500